### DISCUSSION TITLE
There seems to be an error in the display when the whole beat and Ties next bar whole beat

### DISCUSSION THREAD

**Discussion by goodgame365** [CoderLine/alphaTab#1081] [Q&A] [answered]
err is like the img

The PC   is normal, but the mobile  is wrong

![err](https://user-images.githubusercontent.com/17884664/209425149-799b42f4-3aaf-484d-a2aa-94774a412e14.png)

the err file : erro happen the last two  bar 
[err.zip](https://github.com/CoderLine/alphaTab/files/10298135/err.zip)

**Comment by goodgame365** [comment, marked as answer]
Sorry, it's my problem

src\rendering\staves\RenderStaff.ts  

when     public topSpacing: number = 0; then happen this err

**Comment by Danielku15** [comment]
As you found out: such low level customizations on the rendering pipeline are sensitive to cause problems because they need to be aligned with certain placement algorithms. 

There was also https://github.com/CoderLine/alphaTab/issues/910 reported where somebody tried to fiddle with this setting. 

You might want to create a post in the [Ideas ](https://github.com/CoderLine/alphaTab/discussions/categories/ideas) section to talk more detailed about why you would want to configure this setting. There is already [this backlog](https://github.com/CoderLine/alphaTab/issues/1001) item to rework and optimize paddings/margins to reduce unnecessary spacings.

**Reply by goodgame365** [reply]
THank,I'm looking forward this backlog That's exactly what I need
